assets/prototype/application/services/apollo/relations/test/RelationsManager.addRelation.test.ts

```diff
@@ -0,0 +1,75 @@
+import { renderHook, act } from '@testing-library/react-hooks';
```

Comment:
`RelationsManager.addRelation.test.ts` looks a bit strange to me.
What do you say about creating `useRelationsManager` folder on the same level as `useRelationsManager.ts`
and moving this test there as a `addRelation.test.ts` file ?

Reply:
That's a good suggestion

Reply:
and `useRelationsManager.ts` might be renamed to `index.ts` in that folder.

Reply:
Changed